catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_RGB_001.jsonnet

```diff
@@ -0,0 +1,218 @@
+        '[NEON Data Policies and Citation Guidelines]' +
+        '(https://www.neonscience.org/data-samples/data-policies-citation)',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'DOI',
+        description: 'Digital Object Identifier. NEON data that have been released are assigned a DOI.',
+        type: ee_const.var_type.string,
+      },      
+      {
+        name: 'FLIGHT_YEAR',
+        description: 'Year the data were collected',
+        type: ee_const.var_type.int,
+      },
+      {
+        name: 'NEON_DOMAIN',
+        description:
+          'NEON eco-climatic domain code, "D01" to "D20". See ' +
+          '[NEON Field Sites and Domains]' +
+          '(https://www.neonscience.org/field-sites/about-field-sites)',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'NEON_SITE',
+        description: 'NEON four-digit site code. See ' +
+        '[NEON Field Sites](https://www.neonscience.org/field-sites/)',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'NEON_DATA_PROD_ID',
+        description: 'NEON data product identification code. Always set to: "DP3.30010.001"',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'NEON_DATA_PROD_URL',
+        description: 'NEON data product url. Always set to: ' +
+        '[https://data.neonscience.org/data-products/DP3.30010.001]' +
+        '(https://data.neonscience.org/data-products/DP3.30010.001)',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'SENSOR_NAME',
+        description:
+          'Make and model of the camera sensor: ' +
+          '"Phase One D8900", "Phase One IQ180", ' +
+          '"Phase One iX-RS 1000", "Phase One iXM-RS 150F"',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'SENSOR_SERIAL',
+        description:
+          'Serial number of the camera sensor: ' +
+          '"EH021537", "EH021656", "EH021554", "FT010031", ' +
+          '"MM010175", "MM010176", "YC030119", "YC030129"',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'PROVISIONAL_RELEASED',
+        description: 'Whether the data are Provisional or Released. See ' +
+        '[https://www.neonscience.org/data-samples/data-management/data-revisions-releases]' +
+        '(https://www.neonscience.org/data-samples/data-management/data-revisions-releases)',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'RELEASE_YEAR',
+        description: 'If data are released, the year of the NEON Release Tag.',
+        type: ee_const.var_type.int,
+      },
+    ],
+    gsd: [0.1],
+    platform: ['NEON'],
+    instruments: [
+      'Phase One D8900',
+      'Phase One IQ180',
+      'Phase One iX-RS 1000',
+      'Phase One iXM-RS 150F',
+    ],
+    'eo:bands': [
+      {
+        name: 'R',
+        description: 'Red',
+        'gee:units': units.dimensionless,
```

Comment:
for NAIP, we use units.dn

https://github.com/google/earthengine-catalog/blob/main/catalog/USDA/USDA_NAIP_DOQQ.jsonnet